src/main/java/org/hydev/mcpm/client/database/ChecksumImplementation.java

```diff
@@ -0,0 +1,31 @@
+package org.hydev.mcpm.client.database;
+
+import org.hydev.mcpm.client.models.PluginModel;
+import org.hydev.mcpm.client.models.PluginVersion;
+import org.hydev.mcpm.client.models.PluginYml;
+import org.hydev.mcpm.utils.PluginJarFile;
+
+import com.opencsv.CSVWriter;
+
+import java.io.File;
+import java.util.List;
+import java.util.ArrayList;
+import java.io.*;  
+import java.util.Scanner;
+import java.util.jar.JarFile;
+
+import javax.naming.NameNotFoundException;
+import javax.swing.plaf.metal.MetalIconFactory.FileIcon16; 
+
+public class ChecksumImplementation {
```

Comment:
What's this class for btw? What's the difference between this and HashUtils?

Reply:
It shouldn't exist, I'll get rid of it and turn it into a method.